arkfilepicker/src/main/java/space/taran/arkfilepicker/folders/FoldersRepo.kt

```diff
@@ -130,19 +128,31 @@ class FoldersRepo(private val appCtx: Context) {
         withContext(Dispatchers.IO) {
             if (folders.containsKey(root)) {
                 forgetRoot(root)
-                deleteFilesRecursively(root)
+                Log.d(
+                    "folders repo",
```

Comment:
For module logged, let's use constant defined in some common place.

Reply:
Fixed in another PR